suggest/pathogens.txt

```diff
@@ -0,0 +1,296 @@
+206,African Swine Fever
+1,African Trypanosomiasis
+203,Algae
+3,Amebiasis
+221,Amoeba
+290,Anaplasmosis
+242,Animal Die-off
+231,Anisakis
+4,Anthrax
+248,Antimicrobial resistance
+5,Arenavirus
+6,Ascariasis
+7,Aspergillosis
+192,Avian Cholera
+68,Avian Influenza
+283,Avian Influenza H5N1
+293,Avian Influenza H5N2
+289,Avian Influenza H5N6
+292,Avian Influenza H5N8
+287,Avian Influenza H7N2
+299,Avian Influenza H7N8
+271,Avian Influenza H7N9
+8,Babesiosis
+9,Bacterial Meningitis
+232,Bartonella
+207,Bite
+10,Blastomycosis
+205,Blue Ear
+170,Bluetongue
+11,Botulism
+183,Bovine Pleuropneumonia
+226,Bovine TB
+204,Bronchitis/Bronchiolitis
+13,Brucellosis
+12,BSE
+278,Burkholderia cepacia
+22,C. difficile
+14,Campylobacter
+256,Canine distemper
+259,Canine Influenza
+281,Carbapenem-resistant enterobacteriaceae (CRE)
+16,Cat Scratch Fever
+17,Chagas
+18,Chicken Pox
+19,Chikungunya
+107,Chlamydia
+20,Cholera
+177,Chronic Wasting Disease
+202,Chronic/Non-Infectious Disease
+275,Citrobacter freundii
+295,Citrus Canker
+294,Citrus Greening
+27,CJD
+167,Classical Swine Fever
+280,Clostridium perfringens enterotoxin (CPE)
+23,Coccidioidomycosis
+25,Cold
+235,Colitis
+24,Colorado Tick Fever
+219,Conflict
+26,Conjunctivitis
+201,Coronavirus
+158,Crimean-Congo Hemorrhagic Fever
+28,Cryptococcosis
+29,Cryptosporidium
+31,Cysticercosis
+32,Cytomegalovirus
+33,Dengue
+34,Diarrhea
+35,Diphtheria
+36,Diphyllobothriasis
+37,Dracunculiasis
+38,Dysentery
+39,E. coli
+40,Eastern Equine Encephalitis
+41,Ebola
+42,Echinococcosis
+43,Elephantiasis
+303,Elizabethkingia
+296,Emerald Ash Borer
+44,Encephalitis
+273,Enterobacter cloacae
+45,Enterobiasis
+193,Enterococcus
+213,Enterovirus
+288,Enterovirus 68
+218,Environmental
+263,Eosinophilia
+46,Eosinophilic Pneumonia
+178,Epizootic Hemorrhagic Disease
+173,Equine Herpes Virus
+159,Equine Infectious Anemia
+210,Equine Influenza
+47,Fascioliasis
+48,Fasciolopsiasis
+258,Feline immunodeficiency virus
+260,Feline infectious peritonitis
+257,Feline leukemia virus
+254,Feline panleukopenia virus
+216,Fever
+49,Filariasis
+220,Flea-borne disease
+267,Food-related toxin
+50,Foodborne Illness
+57,Foot and Mouth
+135,Foot Fungus
+51,Free Living Amoeba
+270,Fungal Meningitis
+150,Gastroenteritis
+52,Giardiasis
+53,Glanders
+230,Gnathostoma
+54,Gnathostomiasis
+55,Gonorrhea
+264,Guillain Barre
+56,Guinea Worm
+222,H. influenzae
+59,H. pylori
+169,Hand
+58,Hantavirus
+261,Helminths
+151,Hemorrhagic Fever
+60,Hepatitis
+243,Hepatitis A
+244,Hepatitis B
+245,Hepatitis C
+246,Hepatitis D
+247,Hepatitis E
+61,Herpes
+62,Herpes Zoster
+266,HFM-CoxsackieA
+265,HFM-Enterovirus71
+63,Histoplasmosis
+2,HIV/AIDS
+64,Hookworm
+94,Hospital-Related Infection
+65,HPV
+172,Human Bocavirus
+66,Hymenolepiasis
+67,Influenza
+224,Influenza H1N1
+268,Influenza H3N2
+69,Isosporiasis
+187,Japanese Encephalitis
+269,Kawasaki Disease
+272,Klebsiella pneumoniae
+71,Kuru
+301,Kyasanur Forest Disease
+240,La Crosse Encephalitis
+72,Lassa Fever
+73,Legionnaires'
```

Comment:
extra ' ?

Reply:
Taken from https://docs.google.com/spreadsheets/d/1qOnGJQg1gtbmNxAW8c9hC1TvFpIQmSnORD-x-iXrpJ0/edit#gid=1945986228
Google results seem to indicate the ' is apart of the name